byway/src/main/webapp/script.js

```diff
@@ -15,14 +15,144 @@
 /**
  * Adds a random greeting to the page.
  */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
 
-  // Pick a random greeting.
-  const greeting = greetings[Math.floor(Math.random() * greetings.length)];
+window.onload = function() {
```

Comment:
Unless these functions are used by other pages, consider moving these functions to their own new script file that corresponds your new page. Try to keep javascript code organized into separate files used in different places. or you can organize them by what actions they are performing instead, or both. This prevents one mistake affect all pages.

Reply:
done! made seperate js for each page